Yoast/Tests/Files/FileNameUnitTests/ExcludedFile.inc

```diff
@@ -1,5 +1,6 @@
-@codingStandardsChangeSetting Yoast.Files.FileName exclude ExcludedFile.inc
+<!-- Annotation must be on the line 2 as this sniff throws issues on line 1 and PHPCS ignores errors on annotation lines. -->
```

Comment:
Unneeded `the` fore `line 2` (in all files below as well)

Reply:
Fixed in the respective commits & force pushed.